omniNotes/src/androidTest/java/it/feio/android/omninotes/Helpers/HelpersMethods.java

```diff
@@ -0,0 +1,86 @@
+package it.feio.android.omninotes.Helpers;
+
+import android.support.test.espresso.NoMatchingViewException;
+import android.view.View;
+
+import org.hamcrest.Matcher;
+
+import java.security.SecureRandom;
+
+import it.feio.android.omninotes.Screen.TextNote;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.pressImeActionButton;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.PositionAssertions.isLeftOf;
+import static android.support.test.espresso.assertion.PositionAssertions.isRightOf;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+
+public class HelpersMethods extends EspressoMethods {
+    public static boolean isVisible(Matcher<View> matcher) {
+        try {
+            onView(matcher).check(matches(isDisplayed()));
+            return true;
+        } catch (Exception e) {
+            // View is not in hierarchy
+            return false;
+        }
+    }
+
+    public static void clickAction(Matcher<View> matcher) {
+        onView(matcher).perform(click());
+    }
+
+    public static boolean isFirstMatcherToTheLeftOfSecondMatcher(Matcher<View> matcher1, Matcher<View> matcher2) {
```

Comment:
Here you don't check if matcher1 is left of matcher2. You check if element identified with matcher1 is left of the element identified with matcher 2.

Reply:
Done.